packages/clima_ui/lib/screens/location_screen.dart

```diff
@@ -76,11 +74,26 @@ class _LocationScreenState extends State<LocationScreen> {
     final _themeState = context.read(themeStateNotifier);
     final weatherState = useProvider(w.weatherStateNotifierProvider.state);
     final weatherStateNotifier = useProvider(w.weatherStateNotifierProvider);
+    final cityName = useState('');
```

Comment:
This is not needed any more as it appears that `FloatingSearchAppBar` stores the text internally.